dts/bindings/spi/nxp,imx-flexspi.yaml

```diff
@@ -13,3 +13,50 @@ properties:
 
     interrupts:
       required: true
+
+    ahb-base:
+      type: int
+      required: true
+      description: |
+        AHB address for read/write access to serial flash memory.
+
+    ahb-bufferable:
+      type: boolean
+      required: false
+      description: |
+        Enable AHB bufferable write access.
```

Comment:
For these, it might be good to add to the `description` something like see details in the `AHBCR register in UM for more details`